### PR TITLE
Feature/add tooltip component

### PR DESCRIPTION
**Pull request by RishabhC-137**
This pull request resolves several issues mentioned in [Issue #35](https://github.com/DiceDB/playground-web/issues/35):

- Styling Fixes: Aligned styles for “Cleanup in:” and “Commands left:”; updated to “Commands left:” and added tooltips.
- File Cleanup: Removed mockData.json file.
- Variable Update: Changed “blacklist” to “blocklist” in variable names.

### CODE REVIEW

**Comment by RishabhC-137** [comment]
@lucifercr07 can you review this pr?

**Comment by lucifercr07** [comment]
@RishabhC-137 please resolve conflicts.

**Comment by RishabhC-137** [comment]
@lucifercr07 can you check

File: next-env.d.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/// <reference types="next" />
```
Comment (lucifercr07) [inline next-env.d.ts:1]: Why's this file required?

**Review by lucifercr07 — CHANGES_REQUESTED** [review]
@RishabhC-137 can you please add some screenshots for the changes?

**Comment by RishabhC-137** [comment]
@lucifercr07 Hi, I added tooltip (i) icons next to both phrases to provide context:
- "Cleanup in:": "Time remaining for cleanup tasks."
- "Commands left:": "Number of commands you can still execute."

https://github.com/user-attachments/assets/73135864-373b-426e-83bf-155f487b18f0

image

**Comment by RishabhC-137** [comment]
@lucifercr07 can you check

**Pull request merged**